backend/src/main/java/de/otto/platform/gitactionboard/config/security/WebSecurityConfig.java

```diff
@@ -97,7 +99,7 @@ public SecurityFilterChain permitAllFilterChain(
   @Order(2)
   @Configuration
   @ConditionalOnExpression(
-      "T(org.springframework.util.StringUtils).hasText('${BASIC_AUTH_USER_DETAILS_FILE_PATH:}')")
+      "T(org.springframework.util.StringUtils).hasText('${BASIC_AUTH_USER_DETAILS_FILE_PATH:}') or T(org.springframework.util.StringUtils).hasText('${BASIC_AUTH_USER_DETAILS:}')")
```

Comment:
Same as previous comment mentioned for backend/src/main/java/de/otto/platform/gitactionboard/adapters/controller/BasicAuthenticationController.java